e2e/LongHaul/device/Parameters.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Devices.Client;
+using CommandLine;
+
+namespace Microsoft.Azure.IoT.Thief.Device
+{
+    public enum TransportType
+    {
+        Mqtt,
+        Amqp,
+    };
+
+    internal class Parameters
+    {
+        [Option(
+            'a',
+            "ApplicationKey",
+            Required = true,
+            HelpText = "The instrumentation key string for application insights.")]
+        public string ApplicationKey { get; set; }
```

Comment:
Needs env var defaults.

Reply:
I couldn't assign env var as defaults here. It's complaining with [CS0182](https://learn.microsoft.com/en-us/dotnet/csharp/misc/cs0182?f1url=%3FappId%3Droslyn%26k%3Dk(CS0182))